---
layout: default
modal-id: 2
date: 2011-10-11
img: flower.png
alt: image-alt
project-date: October 2011
journal: PLoS ONE
doi: <a href="http://dx.doi.org/10.1371/journal.pone.0025922">10.1371/journal.pone.0025922</a>
authorship: Mid
article: Research
topic: Molecular biology | Bioinformatics | EBV
description: This paper was produced after I left the lab of Professor Alison Sinclair at the University of Sussex, with whom I did my MSc project followed by a short Fellowship.<p> My contribution to this paper was largely bioinformatic, building on work performed during my MSc project in which I'd performed bioinformatic assessments of the EBV genome to try to identify putative ZREs (Zta response elements, sites at which that key lytic cycle transactivator can mediate its effects) primarily using a combination of the webtool <a href="http://alggen.lsi.upc.es/cgi-bin/promo_v3/promo/promoinit.cgi?dirDB=TF_8.3">PROMO</a> and positional weight matrices. During and after my leaving the lab, this approach was further refined and used as the foundation upon which which Kirsty and others built the much greater body of work on the wet lab side of things, experimentally validating the sites through a variety of DNA association assays.<p>Not only did this work display the distribution and diversity of ZREs in the viral genome, but it describes how most lytic cycle genes have one or more ZREs which require methylation for proper Zta interaction, revealing how the virus can target transactivation to otherwise epigenetically silenced genes during the lytic switch.
---
