---
title: "Decombinator: a tool for fast, efficient gene assignment in T-cell receptor sequences using a finite state machine"
collection: publications
permalink: /publication/2013-01-09-decombinator
excerpt: [![Static Badge](https://img.shields.io/badge/pdf-8A2BE2)](http://jamieheather.github.io/files/Thomas_2013_Bioinf_Decombinator.pdf) [![Static Badge](https://img.shields.io/badge/doi-8A2B22)](https://doi.org/10.1093/bioinformatics/btt004) 'Decombinator, for TCRseq annotation'
date: 2013-01-09
venue: 'Bioinformatics'
paperurl: 'http://jamieheather.github.io/files/Thomas_2013_Bioinf_Decombinator.pdf'
---

This was my first publication in the Chain lab at UCL, being the PhD work of my mathematician labmate Niclas Thomas. Nic wrote the first version of Decombinator, the lab's script for annotating TCR rearrangements from deep sequencing data - using the key innovation of using unique tag sets to populate Aho Corasick tries to identify V and J genes. My primary contribution was to develop capacity for processing alpha chain data to complement the beta chain functionality that already existed.

[![Static Badge](https://img.shields.io/badge/pdf-8A2BE1)](http://jamieheather.github.io/files/Thomas_2013_Bioinf_Decombinator.pdf) [![Static Badge](https://img.shields.io/badge/doi-8A2B22)](https://doi.org/10.1093/bioinformatics/btt004)
